tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/genitals.tsx

```diff
@@ -151,6 +157,7 @@ export const breasts_color: Feature<string[]> = {
 
 export const breasts_emissive: Feature<boolean[]> = {
   name: 'Breast Emissives',
+  description: 'Emissive parts/layers glow in the dark.',
```

Comment:
```suggestion
  description: 'Emissive parts glow in the dark.',
```
You get the idea. There are a bunch of these